### DISCUSSION TITLE
Welcome to crypto-wallet-bruteforce Discussions!

### DISCUSSION THREAD

**Discussion by niyankhadka** [niyankhadka/crypto-wallet-bruteforce#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Goken123** [comment]
Hey how do we widthraw it

**Reply by niyankhadka** [reply]
If the wallet balance is found then, you can see the found address mnemonic phrase and the balance on txt file. For that, you need to go to /hasBalance/hasBalance-xxx.txt file and you can see all the details there.
